lms/static/sass/discussion/_layouts.scss

```diff
@@ -31,4 +31,8 @@
 
 .discussion-column {
     min-height: 500px;
+
+    .new-post-article {
```

Comment:
Hm. Just FYI, @andy-armstrong, the button styles for the edit post form are broken for me: 

![](https://i.bjacobel.com/20160901-fmf45.png)

I would like to consolidate the two forms. I'll look into how much work that will be.